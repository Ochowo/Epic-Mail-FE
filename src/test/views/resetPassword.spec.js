import React from 'react';
import { shallow } from 'enzyme';
import Reset from '../../views/resetpassword/resetPassword';

const props = {};
const setUp = () => {
  const component = shallow(<Reset {...props} />);
  return component;
};
describe('Reset password component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    component.setProps({ isAuthenticated: true });
    expect(component).toMatchSnapshot();
  });

  it('should call the loginForm method', () => {
    const FormSpy = jest.spyOn(component.instance(), 'loginForm');
    const form = component.find('#logId');
    expect(form.length).toBe(1);
    form.simulate('click');
    component.instance().loginForm();
    expect(FormSpy).toBeCalled();
  });
});
