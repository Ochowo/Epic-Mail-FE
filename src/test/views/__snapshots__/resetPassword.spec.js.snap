// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset password component should render without errors 1`] = `ShallowWrapper {}`;
